developer_manual/core/theming.rst

```diff
@@ -1,166 +1,144 @@
+
+.. note:: 
+   When overriding the favicon, make sure your custom theme includes and override for both ``owncloud/apps/core/img/favicon.svg`` and ``owncloud/apps/core/img/favicon.png``, to cover any future updates to favicon handling.
+
+How to Override the Default Colors
+----------------------------------
+
+To override the default style sheet, create a new CSS style sheet in your theme, in the theme’s ``css`` directory, called ``styles.css``.
+
+How to Override Translations
 ----------------------------
-With a web-developer tool like Mozilla-Inspector, you also get easily displayed the color of the background you clicked on.
-On the top of the login page you can see a case- distinguished setting for different browsers:
-
-.. code-block:: css
-
-  /* HEADERS */
-  ...
-  body-login {
-    background: #1d2d42; /* Old browsers */
-    background: -moz-linear-gradient(top, #33537a 0%, #1d2d42  100%); /* FF3.6+ */
-    background: -webkit-gradient(linear, left top, left bottom, color-stop(0%,#F1B3A4), color-stop(100%,#1d2d42)); /* Chrome,Safari4+ */
-    background: -webkit-linear-gradient(top, #33537a 0%,#1d2d42 100%); /* Chrome10+,Safari5.1+ */
-    background: -o-linear-gradient(top, #33537a 0%,#1d2d42 100%); /* Opera11.10+ */
-    background: -ms-linear-gradient(top, #33537a 0%,#1d2d42 100%); /* IE10+ */
-    background: linear-gradient(top, #33537a 0%,#1d2d42 100%); /* W3C */
-  }
-
-The different background-assignments indicate the headers for a lot of different browser types. What you most likely want to do is change the #35537a (lighter blue) and #ld2d42 (dark blue) color to the colours of our choice. In some older and other browsers, there is just one color, but in the rest showing gradients is possible.
-The login page background is a horizontal gradient. The first hex number, #35537a, is the top color of the gradient at the login screen. The second hex number, #ld2d42 is the bottom color of the gradient at the login screen.
-The gradient in top of the normal view after login is also defined by these CSS-settings, so that they take effect in logged in situation as well.
-Change these colors to the hex color of your choice:
-As usual:
 
-* the first two figures give the intensity of the red channel,
-* the second two give the green intensity and the
-* third pair gives the blue value.
-
-Save your CSS-file and refresh to see the new login screen.
-The other major color scheme is the blue header bar on the main navigation page once you log in to ownCloud.
-This color we will change with the above as well.
-Save the file and refresh the browser for the changes to take effect.
+.. versionadded 8.0
 
+You can override the translation of any string in your theme. 
+To do so:
 
-How to change translations
-==========================
-.. versionadded 8.0
+1. Create the ``l10n`` folder inside your theme, for the app that you want to override.
```

Comment:
If you want to overwrite a translation of the files app, you would have to create the locale file inside `owncloud/apps/theme-example/apps/files/l10n/de_DE.js`. The structure is the same like for images, just mimic the original file location inside your theme.

We had a lot of exceptions earlier, but with the new theming support we don't make any exceptions, everything follows one unified way.